lang/levels/standard/text_fre.po

```diff
@@ -0,0 +1,841 @@
+msgstr "Les barbares que vous avez tués sont revenus sous forme de fantômes et ont rejoint votre armée. Je vous suggère de tuer tous les barbares que vous pouvez trouver ? "
+
+#. Ghost in the Machine (lv458)
+#: guitext:95
+msgctxt "In-game message"
+msgid "The Wizards will keep sending Barbarians in ever greater numbers, but it is all a distraction. They will use this time to grow beyond your might. Keep their power in check by regularly attacking them. "
+msgstr "Les sorciers continueront d'envoyer des barbares en nombre toujours plus grand, mais tout cela n'est qu'une distraction. Ils utiliseront ce temps pour grandir au-delà de vos forces. Gardez leur pouvoir sous contrôle en les attaquant régulièrement. "
+
+#. Ghost in the Machine (lv458)
+#: guitext:96
+msgctxt "In-game message"
+msgid "With each Monk killed, the Ghosts gain in strength! Keep killing Monks as well as Barbarians. "
+msgstr "Avec chaque Moine tué, les fantômes gagnent en force ! Continuez à tuer des moines ainsi que des barbares. "
+
+#. Ghost in the Machine (lv458)
+#: guitext:97
+msgctxt "In-game message"
+msgid "There are no more Monks so your Ghosts are as strong as they can get. They should be able to do some serious damage to the yellow Wizards? Dispose of your imps to initiate the attack. "
+msgstr "Il n'y a plus de moines donc vos fantômes sont aussi forts que possible. Ils devraient être capables de faire de sérieux dégâts aux sorciers jaunes ? Débarrassez-vous de vos larbins pour lancer l'attaque. "
+
+#. Ghost in the Machine (lv458)
+#: guitext:98
+msgctxt "In-game message"
+msgid "Your Ghosts are being transported to the yellow Hero stronghold! Destroy their heart and this realm is yours."
+msgstr "Vos fantômes sont transportés vers la forteresse jaune des héros ! Détruisez leur coeur et ce royaume est à vous."
+
+#. Ghost in the Machine (lv458)
+#: guitext:99
+msgctxt "In-game message"
+msgid "These Ghosts are at maximum power. Surely the wizards don't stand a chance now?"
+msgstr "Ces fantômes sont à leur puissance maximale. Les sorciers n'ont sûrement plus aucune chance maintenant ?"
+
+#. Ghost in the Machine (lv458)
+#: guitext:100
+msgctxt "In-game message"
+msgid "Your Ghosts have failed and paid the price for it! Luckily there's always more death? When you have regained enough Ghosts, eliminate your imps to try again."
+msgstr "Vos fantômes ont échoué et en ont payé le prix ! Heureusement qu'il y a toujours plus de mort ? Lorsque vous aurez récupéré suffisamment de fantômes, éliminez vos larbins pour réessayer."
+
+#. Ghost in the Machine (lv458)
+#: guitext:101
+msgctxt "In-game message"
+msgid "This is taking too long. Let's see what happens."
+msgstr "Cela prend trop de temps. Voyons ce qui se passe."
+
+#. Ghost in the Machine (lv458)
+#: guitext:102
+msgctxt "In-game message"
+msgid "\"Your message could be posted here. Victory!"
+msgstr "\"Votre message pourrait être posté ici. Victoire !"
```

Comment:
A quoi sert le "\" ici ?

Reply:
c'est pour eviter que le second " soit interprété. cherche "escape characters sur internet"

Reply:
Well, it did not need the second colon. I removed it now both in english and in french.